### PR TITLE
Some minor tweaks to improve the integration

### PR DESCRIPTION
**Pull request by alanmcgovern**

### CODE REVIEW

File: LVST/Program.cs [hunk 1]
```diff
@@ -113,6 +113,8 @@ private static async Task<bool> FindAndUseChromecast()
         private static async Task<Stream> StartTorrenting(Options cliOptions)
         {
             var engine = new ClientEngine();
+            await engine.RegisterDhtAsync (new MonoTorrent.Dht.DhtEngine (MonoTorrent.Dht.Listeners.DhtListenerFactory.CreateUdp (5555)));
```
Comment (alanmcgovern) [inline LVST/Program.cs:116]: This is a bit crappy... but... :)

Comment (mfkl) [reply LVST/Program.cs:116]: Don't know much about DHT but I trust you :-)
Can make the port configurable later. Been thinking of a `LibVLCSharp.MonoTorrent` nuget.

**Comment by mfkl** [comment]
Thanks! 👍

File: LVST/Program.cs (hunk 1 above)
Comment (alanmcgovern) [reply LVST/Program.cs:116]: That works for me! The port isn't super important as long as it doesn't clash with an already in use port. 

You need to forward it using upnp or nat-pmp (or manually) for some aspects of the DHT capabilities, but lookups for peers should work just fine without an open port so this is a net win

**Pull request merged**